### ISSUE TITLE
Route Sonobus to virtual device on macOS

### ISSUE THREAD

**Issue by fiveohfiveoh** [sonosaurus/sonobus#160]
Great project! Can SonoBus route audio to a virtual audio device on macOS, like Source Nexus does? https://www.source-elements.com/products/source-nexus/ 

Use case would be: Route audio out of a DAW, into "SonoBus Virtual Device", then select "SonoBus Virtual Device" as the audio source in Zoom, OBS etc. 

I don't see this functionality in the current release, just wondered if it's on the roadmap or if it would be possible to do in the future.

**Comment by essej** [comment]
No, it does not include any virtual audio device functionality. Have you tried using BlackHole for that part? https://existential.audio/blackhole/

**Comment by llspalex** [comment]
Thanks for the swift response. Yes I have tried BlackHole, but it requires fiddling with aggregate deviced and outputting to that. I really like simplicity of keeping my setup as-is and just whacking a plugin (like Source Nexus) on to my main track and then being able to pull that signal into wherever I want via virtual device. Also means I can more easily send just specific tracks to the virtual device via a bus.

**Comment by fiveohfiveoh** [comment]
Thanks for the swift response. Yes I have tried BlackHole, but it requires fiddling with aggregate devices and outputting to that. I really like simplicity of keeping my setup as-is and just whacking a plugin (like Source Nexus) on to my main track and then being able to pull that signal into wherever I want via virtual device. Also means I can more easily send just specific tracks to the virtual device via a bus.

**Comment by essej** [comment]
So you are just looking for a free version of source-nexus? Are there any SonoBus features you actually want here?

**Comment by fiveohfiveoh** [comment]
> So you are just looking for a free version of source-nexus?
Yes. 

SonoBus already seems to have many of the ingredients ie. sending an audio signal out from a DAW to somewhere else. I just wondered how involved it would be to make that "somewhere else" be a virtual device on the host machine.
